src/app/header/header.component.html

```diff
@@ -0,0 +1,6 @@
+<div id="logo"></div>
```

Comment:
This should probably be an <a> too which leads to home

you might want to keep the id here.. but be more specific such as id="header-logo"